_scripts/installer/install.sh

```diff
@@ -61,7 +61,11 @@ VERSION="$(get_latest_version "${HELM_VERSION_URL}")"
 HELM_ZIP="helm-${VERSION}-${PLATFORM}-${ARCH}.zip"
 
 echo "Downloading ${HELM_ZIP} from Bintray..."
-curl -Ls "${HELM_BIN_URL_BASE}/${HELM_ZIP}" -O
+if [[ $(which wget) ]]; then
```

Comment:
This isn't quite the logic you want... you want something like:

`if command -v wget &> /dev/null; then`

This way you're specifically dealing with exit codes. See: http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script